CMakeLists.txt

```diff
@@ -18,6 +13,18 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=0")
 # We require a C++11 compliant compiler
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# supress warning of ray_shared
+cmake_policy(SET CMP0068 OLD)
+
+# ray build options
+option(RAY_BUILD_PYTHON
+    "build python library"
+    ON)
+
+option(RAY_BUILD_JAVA
+    "build java library"
```

Comment:
Current building process supports building java and python at the same time. (`./build.sh -l java,python`) Will this PR support this?

Reply:
yes. you can run   `cmake .. -DRAY_BUILD_JAVA=ON` to compile java language libraries